### PR TITLE
Add section for kubectl to dlrs tutorial

### PR DESCRIPTION
**Pull request by bd-dean**

### CODE REVIEW

**Comment by krisdale** [comment]
@bd-dean Have you or someone else done the tech validation of the new section to confirm all works as advertised?

**Comment by bd-dean** [comment]
Tech review completed, all steps verified.

**Comment by krisdale** [comment]
@MCamp859 Would you please provide editorial review on this PR? Tech review is complete per @bd-dean  Thanks!

File: source/clear-linux/tutorials/dlrs/dlrs.rst [hunk 1]
```diff
@@ -196,6 +196,40 @@ We used `flannel`_ as the network provider for these tests. If you
 prefer a different network layer, refer to the Kubernetes
 `networking documentation`_ for setup.
 
+Kubectl
+=======
+
+You can use kubectl to run commands against your Kubernetes cluster.  Refer to the `kubectl overview`_ for details on syntax and operations. Once you have a working cluster on Kubernetes, you can use the following YAML script to start a pod with a simple shell script, and keep the pod open.
```
Comment (MCamp859) [inline source/clear-linux/tutorials/dlrs/dlrs.rst]: Change "you can use the following YAML" to "use the following yaml"

File: source/clear-linux/tutorials/dlrs/dlrs.rst [hunk 2]
```diff
@@ -196,6 +196,40 @@ We used `flannel`_ as the network provider for these tests. If you
...
+
+#. Copy this example.yaml yaml script to your system:
```
Comment (MCamp859) [inline source/clear-linux/tutorials/dlrs/dlrs.rst]: Delete 2nd yaml in this line.

File: source/clear-linux/tutorials/dlrs/dlrs.rst [hunk 3]
```diff
@@ -196,6 +196,40 @@ We used `flannel`_ as the network provider for these tests. If you
...
+          args: [ "while true; do sleep 30; done" ]
+
+#. Execute the script with kubectl:
+
+   .. code-block:: bash
+
+      kubectl apply –f <path-to-yaml-file>/example.yaml
+
+
+This will open a single pod. More robust solutions would create a deployment or inject a python script or larger shell script into the container.
```
Comment (MCamp859) [inline source/clear-linux/tutorials/dlrs/dlrs.rst]: Change to: "This script opens a single pod."  2nd sentence is OK as-is.

File: source/clear-linux/tutorials/dlrs/dlrs.rst (hunk 1 above)
Comment (bd-dean) [reply source/clear-linux/tutorials/dlrs/dlrs.rst]: done!

File: source/clear-linux/tutorials/dlrs/dlrs.rst (hunk 2 above)
Comment (bd-dean) [reply source/clear-linux/tutorials/dlrs/dlrs.rst]: deleted!

File: source/clear-linux/tutorials/dlrs/dlrs.rst (hunk 3 above)
Comment (bd-dean) [reply source/clear-linux/tutorials/dlrs/dlrs.rst]: Changed!

**Comment by MCamp859** [comment]
@bd-dean, LGTM! @krisdale OK to merge.

**Pull request merged**